### ISSUE TITLE
NNS shows no items when returning to nabu.ca after its first connection 

### ISSUE THREAD

**Issue by ontheslab** [GryBsh/NabuNetworkEmulator#14]
Loving the working nabu.ca as I'm currently testing multiple MAME instances on different platforms at once.

Server is a Pi3B running "bullseye" 64bit install, running since 1.1 (I think).

Steps I take and when it fails (I thought it was a concurrent connection issue but it happens on single connection as well):

Connect from MAME as normal.
![image](https://github.com/GryBsh/NabuNetSim/assets/19336841/cfe46cad-b97b-4320-8325-182dff43bcb3)

Select NABU.ca menu loads fine - first load.
![image](https://github.com/GryBsh/NabuNetSim/assets/19336841/aaea3243-6f2b-4e08-a723-20adca756972)

Run Cloud CP/M - all fantastic! (using it to easy test 80col / 40col MAMA)
![image](https://github.com/GryBsh/NabuNetSim/assets/19336841/d8b161da-f9b2-498a-b808-9bc26d538257)

Using multiple "drives" in Cloud CP/M.

"F3" to reset and return to Menu - all good!
![image](https://github.com/GryBsh/NabuNetSim/assets/19336841/6585d23f-b55f-4d9f-9cf9-5ecee756aaa9)

Scratch that, I was going to say / show you that it would load a "blank" nabu.ca menu on return, but its crashed after just running another main menu item.

Just showing adapter failure now and looks like looping on the server..

If I shutdown MAME and restart - comms with the server seems to restart but now the NABU.ca menu is blank.

![image](https://github.com/GryBsh/NabuNetSim/assets/19336841/0627f662-3df5-48a3-9683-707854555bad)

 Long story short I have to "break" the server and restart it.

![image](https://github.com/GryBsh/NabuNetSim/assets/19336841/593fca86-1baa-4abd-8da0-18454f25238e)

![image](https://github.com/GryBsh/NabuNetSim/assets/19336841/07e4a7f2-c14d-4c05-9b77-67f3e69f7a7a)

I hope that makes sense?

[logsnip.log](https://github.com/GryBsh/NabuNetSim/files/14966934/logsnip.log)

**Comment by ontheslab** [comment]
So far so good! I can't break it yet! ;-)
